android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -645,6 +779,101 @@ void createTouchScalar(ImageResolution resolution, DisplayMetrics displayMetrics
         return motionEventList;
     }
 
+    public VideoStreamingParameters getLastCachedStreamingParameters() {
+        return parameters;
+    }
+
+    private List<VideoStreamingCapability> getSupportedCapabilities(
+            List<VideoStreamingRange> ranges,
+            VideoStreamingCapability rootCapability
+    ){
```

Comment:
After Aligning changes with the iOS library there are some cases that were not accounted for or not aligned between the two libraries. Mainly these changes are to decide what it means if the developer set a null range or a range where the Resolutions are (0, 0) and how to handle these cases.

1) If both `supportedPortraitStreamingRange` and `supportedLandscapeStreamingRange` are not set or set to `null` by the developer, then the mobile library will support all `VideoStreamingCapabilities` returned by the module. This is done to ensure that streaming works as intended for developers who don't update the library to add the supportedPortraitStreamingRange and supportedLandscapeStreamingRange parameters.
2) To disable either `supportedPortraitStreamingRange` or `supportedLandscapeStreamingRange`, the developer has to set a `VideoStreamingRange` with the `minimumResolution` and `maximumResolution` set to 0 width and 0 height. If the developer chooses to disable both `supportedPortraitStreamingRange` and `supportedLandscapeStreamingRange`, video will not stream.
3) If `supportedPortraitStreamingRange` is not set or was set to `null` then the library will assume that all `VideoStreamingCapabilities` with a portrait aspect ratio are supported. The same is true if `supportedLandscapeStreamingRange` is not set; the library will assume that all `VideoStreamingCapabilities` with a landscape aspect ratio are supported.

For point 3) please refer to the iOS implementation on how to determine if a capability is a portrait or landscape capability
matchVideoCapability and isCapability in this [file](https://github.com/smartdevicelink/sdl_ios/blob/cb3d5b15a971f46b66e9738fc06b4bc3f98aac8b/SmartDeviceLink/private/SDLStreamingVideoLifecycleManager.m)
as well as SDLImageResolutionKind in this [file](https://github.com/smartdevicelink/sdl_ios/blob/cb3d5b15a971f46b66e9738fc06b4bc3f98aac8b/SmartDeviceLink/private/SDLImageResolution%20%2B%20StreamingVideoExtensions.m)